backend/inspirehep/records/marshmallow/literature/ui.py

```diff
@@ -210,6 +211,25 @@ def get_citation_pdf_urls(self, data):
                 urls.append(url)
         return urls or missing
 
+    def get_datasets(self, data):
+        dataset_links = []
+        dataset_schema_to_url_link_prefix_map = {
+            "hepdata": "https://www.hepdata.net/record/",
+        }
+        dataset_schema_to_description_map = {"hepdata": "HEPData"}
+        all_links = get_value(data, "external_system_identifiers", [])
```

Comment:
You could use `get_values_for_schema`

i.e.

```python
get_values_for_schema(data['external_system_identifiers'], 'HEPData')
```

Reply:
It will work for now, but what if we'll add other dataset sources?